Jenkinsfile

```diff
@@ -77,6 +83,8 @@ pipeline {
     environment {
         KUBECONFIG = "$workspace/.kube/config"
         scannerHome = tool name: 'Sonarqube-certs-PROD', type: 'hudson.plugins.sonar.SonarRunnerInstallation'
+        PATH = "/opt/maker/tools/aws:$PATH"
+        TAG = "${sh(script: "echo `date +%g%m.$GIT_COMMIT`", returnStdout: true).trim()}"
```

Comment:
This tag does not correspond to the tag that is used when pushing the image to ECR.